domain/src/main/java/org/oppia/android/domain/survey/SurveyGatingController.kt

```diff
@@ -66,7 +66,7 @@ class SurveyGatingController @Inject constructor(
     val currentTimeStamp = oppiaClock.getCurrentTimeMs()
     val showNextTimestamp = lastShownTimestampMs + gracePeriodMillis
 
-    return currentTimeStamp > showNextTimestamp || currentTimeStamp == showNextTimestamp
+    return currentTimeStamp >= showNextTimestamp
```

Comment:
Any changes to SurveyGatingControllerTest to cover this?

Reply:
While the change here is mainly syntactic and in a private method, I have added some extra test cases to distinguishe between. `==` and `>` conditions.

Reply:
Thanks. Even behaviors in private methods often can be corresponded to public behaviors (as your tests demonstrate), so it's good to cover them when possible.